src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const people = table.querySelector('tbody');
+const title = table.tHead.querySelectorAll('th');
+
+table.tHead.addEventListener('click', (e) => {
+  const index = [...title].indexOf(e.target);
```

Comment:
you can use .cellIndex
https://learn.javascript.ru/dom-navigation#dom-navigation-tables